### PR TITLE
Mention mode line support for org clock in docs

### PR DESCRIPTION
**Pull request by swaroopch**
Adding documentation.

### CODE REVIEW

File: layers/+emacs/org/README.org [hunk 1]
```diff
@@ -138,6 +139,15 @@ the following snippet. Note that this may have unintended consequences until
     (push (org-projectile:todo-files) org-agenda-files))
 #+END_SRC
 
+** Mode line support
+To temporarily enable mode line display of org clock, press ~SPC t m c~.
+
+To permanently enable mode line display of org clock, add this snippet to your ~dotspacemacs/user-config~ function:
```
Comment (TheBB) [inline layers/+emacs/org/README.org]: This is code, not keys. Surround with `=` instead of `~` please.

Comment (swaroopch) [inline layers/+emacs/org/README.org]: @TheBB I have made the change.

**Comment by swaroopch** [comment]
@TheBB _bump_ - what's the next step? :)

**Comment by TheBB** [comment]
Thanks! Cherry-picked in develop. I filled the paragraph for you. You can safely delete your branch.

**Comment by swaroopch** [comment]
> Thanks! Cherry-picked in develop.

Thanks!

> I filled the paragraph for you.

Thanks again!

**Pull request closed without merging**